backend/unitTests/loginTest.py

```
@@ -0,0 +1,75 @@
+
+BASE_URL = "http://localhost:8000"
+
+def test_admin_login():
+    """Test default admin login"""
+    admin_data = {
+        "username": "admin",
+        "password": "password123"
+    }
+    
+    try:
+        response = requests.post(f"{BASE_URL}/api/auth/login", json=admin_data)
```

Ruff (0.12.2)

Probable use of `requests` call without timeout

(S113)